apps/05Mastermind.js

```diff
@@ -6,7 +6,7 @@ var prompt = require('prompt');
 prompt.start();
 
 var board = [];
-var solution = '';
+var solution = 'abcd';
```

Comment:
This isn't good.  This is resulting in a long solution.  Your program did this when I played it:

```
You ran out of turns! The solution was abcdfhbc
```